interfaces/builtin/password.go

```diff
@@ -0,0 +1,42 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const passwordConnectedPlugAppArmor = `
+/usr/bin/passwd ix,
+/usr/sbin/adduser ix,
+/usr/sbin/useradd ix,
+/usr/sbin/chpasswd ix,
+
+/var/lib/extrausers/{,**} rw,
```

Comment:
I think you are going to need 'rwk' here. If you don't in testing, I suspect you may be hitting https://bugs.launchpad.net/apparmor/+bug/1658219.

Please add the comment:
```
# Only allow modifying the non-system extrausers database
```